src/cloud/benchflow/sender/sender.go

```diff
@@ -164,8 +164,10 @@ func consumeFromTopic(t TransformerSetting) {
 				continue
 				}
 			fmt.Println(t.Topic+" received: "+msg.Minio_key)
+			minioKeys := strings.Split(msg.Minio_key, ",")
```

Comment:
@VincenzoFerme This works regardless, because if there's only one key rather than a coma separated list it will just read that one key.